R/create_interactive_report.R

```diff
@@ -119,6 +120,13 @@ create_interactive_report <-
       file.copy(survey_files, to = fs::path(output_dir, "survey"), overwrite = TRUE, recursive = TRUE)
     }
 
+    # real estate path ---------------------------------------------------------
+
+    real_estate_file <- fs::path(output_dir, "real_estate") |>
+      fs::dir_info() |>
+      dplyr::filter(grepl("es_.*_de", path)) |>
+      dplyr::pull(path)
```

Comment:
This should probably be moved into the `if (length(list.files(real_estate_dir)) > 0)` block a little later

Reply:
Also, this is going to need to be changed since it is currently generating the full path to the users local version of the file. 

I guess we will need a relative one